src/pages/Menu.js

```diff
@@ -1,8 +1,14 @@
 // import { useEffect, useState } from "react";
 import MenuItem from "../components/MenuItem";
 
-export default function Menu({ items }) {
+export default function Menu({ items, updateItemQuantity }) {
   return items.map((item) => {
-    return <MenuItem key={item.id} item={item} />;
+    return (
+      <MenuItem
+        key={item.id}
+        item={item}
+        updateItemQuantity={updateItemQuantity}
```

Comment:
Another nice feature for the future could be that the chosen quantity stays fixed, so you can edit and then it sets to 0 once you confirm your order. Right now it sets to 0 when I go to the shopping cart, so if say I want to increase my order of Bungo Bbang I have to go back and forth to see how many I had and then increase. Just a suggestion.

Reply:
that's exactly what I still need to fix :D Don't know how yet.